app/views/spree/admin/products/new.html.haml

```diff
@@ -62,7 +62,8 @@
                   "question-mark-with-tooltip-animation" => true, 
                   key: "'js.admin.unit_price_tooltip'"}
               %br/
-              = f.text_field :price, {"class" => 'fullwidth', "disabled" =>  true, "ng-model" => "unit_price"}
+              %input{ "type" => "text", "id" => "variant_unit_price", "name" => "variant[unit_price]",
+                  "class" => 'fullwidth', "disabled" =>  true, "ng-model" => "unit_price"}
```

Comment:
Wouldn't you have got the same result with:
```haml
= f.text_field :unit_price, {"class" => 'fullwidth', "disabled" =>  true, "ng-model" => "unit_price"}
```
:question:

Reply:
Actually no, because `unit_price` is not a method of `Product`
```
undefined method `unit_price' for #<Spree::Product:0x00007fe23c277ab0>
```
(it's a readonly field, computed from the others)

Reply:
Ah, that makes sense. :+1: